content/docs/dvclive/api-reference/set_step.md

```diff
@@ -0,0 +1,54 @@
+# dvclive.set_step()
+
+Signals that the current step has ended and updates `step` with the given value.
+
+```py
+def set_step(step: int):
+```
+
+#### Usage:
+
+```py
+import dvclive
+
+for step in [0, 10, 20]:
+    dvclive.set_step(step)
```

Comment:
The link checker is seeing this as a link for some reason, that's on us and can be ignored.

Reply:
Haha

```js
[0, 10, 20]:
    dvclive.set_step(step)
```
looks like an md reference link 🤦 

Ideally the link checker should ignore everything inside md code blocks, I think.

Reply:
p.s. a quick fix would be to start that regex with `^\[`.